Cargo.toml

```diff
@@ -34,7 +34,7 @@ fsevent = "^0.2.17"
 fsevent-sys = "^0.1.3"
 
 [target.'cfg(windows)'.dependencies]
-winapi = "^0.2.5"
+winapi = "^0.3.5"
 kernel32-sys = "^0.2.1"
```

Comment:
When upgrading winapi to "0.3", `kernel32-sys` should be dropped.

Reply:
ack, thanks